src/Chirp.Core/Classes/Follow.cs

```diff
@@ -0,0 +1,10 @@
+namespace Chirp.Core.Classes;
+
+public class Follow
```

Comment:
is it necessary to have FollowerId and FollowedId? Wouldn't having reference to the authors suffice?

Reply:
The ID's are needed, since they establish a many to many relationship and serve as foreign keys for the Follow relation. The composite key: FollowerID and FollowedID uniquely identify the relationship between authors